src/pages/CreateEventPage/CreateEventPage.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from 'react';
+import { CreateEventWrapper, Form } from './styled';
+import { formatISO, format } from 'date-fns';
+import { createEvent } from '../../api/events';
+import TextField from '../../components/TextField';
+import DateField from '../../components/DateField';
+import TimeField from '../../components/TimeField';
+import SelectField from '../../components/SelectField';
+import TextAreaField from '../../components/TextAreaField';
+import Alert from '../../components/Alert';
+
+function CreateEvent() {
+  const [name, setName] = useState('');
+  const [image, setImage] = useState('');
+  const [imageAlt, setImageAlt] = useState('');
+  const [date, setDate] = useState(format(Date.now(), 'yyyy-MM-dd'));
+  const [starttime, setStarttime] = useState(format(Date.now(), 'HH:mm'));
+  const [endtime, setEndtime] = useState(format(Date.now(), 'HH:mm'));
+  const [description, setDescription] = useState('');
+  const [eventType, setEventType] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [createWasSuccess, setCreateWasSuccess] = useState(false);
+  const [createFailed, setCreateFailed] = useState('');
```

Comment:
This is no longer a boolean, so `createError` or something similar makes more sense in my opinion.